Update Chef Infra Server Release Notes for 13.1.13 (#2204)
chef/chef-web-docs: chef_master/source/release_notes_server.rst

Chef Server acts as a hub for configuration data by storing cookbooks, the policies that are applied to nodes, and metadata that describes each registered node that is managed by the chef-client.

What's New in 13.1.13
=====================================================

Improvements/Bug Fixes
-----------------------------------------------------

> * ``_status`` endpoint reports healthy even if ``data_collector`` is down not causing unnecessary failovers 

Reviewer: ```suggestion
* The ``_status`` endpoint now reports healthy even if the ``data_collector`` is down which will no longer cause unnecessary failovers.
```

> * Data collector proxy-header X-Forwarded for is set as expected

Reviewer: ```suggestion
* Data collector proxy-header X-Forwarded is set as expected.
```

> * ``chef-server-ctl`` is no longer installed in the user path only the appbundled version is installed in the user path

Reviewer: ```suggestion
* ``chef-server-ctl`` is no longer installed in the user path. Now only the appbundled version is installed in the user path.
```

> * Fixes issue with Chef Support Zendesk sign-ins when a first name is not set in Hosted Chef

Reviewer: ```suggestion
* Fixed an issue with Chef Support Zendesk sign-ins when a first name is not set in Hosted Chef.
```

> * Added support for running the Chef Infra Server on Red Hat Enterprise Linux 8

Reviewer: ```suggestion
* Added support for running the Chef Infra Server on Red Hat Enterprise Linux 8.
```

> * Improvements to ``chef-server-ctl gather-logs``

Reviewer: ```suggestion
* ``chef-server-ctl gather-logs`` was updated with the following improvements:
```

>  * Add AWS to known platforms

Reviewer: RST requires a space before a nested bullet point item
```suggestion

 * Add AWS to known platforms
```

 * Add AWS Native Chef Server info
 * Add elasticsearch info
 * Switched compression from bzip2 to gzip

Deprecation Notice
-----------------------------------------------------

> * SLES 11 is no longer supported per our [platform policy](https://docs.chef.io/platforms.html#platform-end-of-life-policy), as upstream support ended in March of this year.

Reviewer: ```suggestion
* SLES 11 is no longer supported per our `platform policy </platforms.html#platform-end-of-life-policy>`_, as upstream support ended in March of this year.
```
